ctrlf.el

```diff
@@ -738,6 +750,7 @@ And self-destruct this hook."
 
 (defun ctrlf--minibuffer-exit-hook ()
   "Clean up CTRLF from minibuffer and self-destruct this hook."
+  (setq ctrlf--minibuffer nil)
```

Comment:
This is that cleanup. Maybe this belongs in the `ctrlf--finalize`? I wasn't entirely sure. We need this because it's insufficient to check `ctrlf--active-p` if some logic is being performed before the start of a search.

Reply:
I think where you have it is fine. `ctrlf--finalize` is somewhat more of a hack, because it runs from `post-command-hook` and then removes itself right away. Ideally we wouldn't have to put anything there.

Reply:
Good to know.